tests/API/XCCDF/unittests/test_generate_fix_ansible_vars_ds-tailoring.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xccdf:Tailoring xmlns:xccdf="http://checklists.nist.gov/xccdf/1.2" id="xccdf_scap-workbench_tailoring_default">
+  <xccdf:benchmark href="/home/matus/test_generate_fix_ansible_vars_ds.xml"/>
+  <xccdf:version time="2018-01-10T09:59:11">1</xccdf:version>
+  <xccdf:Profile id="xccdf_com.example.www_profile_test_ansible_vars_tailored" extends="xccdf_com.example.www_profile_test_ansible_vars">
+    <xccdf:title xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en-US" override="true">xccdf_test_profile [CUSTOMIZED]</xccdf:title>
+    <xccdf:description xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en-US" override="true">This profile is for testing.</xccdf:description>
+    <xccdf:select idref="xccdf_com.example.www_rule_default2" selected="false"/>
```

Comment:
Can you also refine the value used by `www_rule_default2`, so that we check if variable in playbook is tailored also.

Reply:
Done in the commit 6a382b0 for variable `www_value_val1` as it is used in the rule `www_rule_default1`, rule `www_rule_default2` is not selected in tailoring.